### PR TITLE
fixes #23

### LINKED ISSUE #23: Click on an header pass click through underneath
When using `StickyRecyclerHeadersTouchListener` the click event is passed to underneath views.

### PR DESCRIPTION
**Pull request by forcewill**

### CODE REVIEW

File: library/src/main/java/com/timehop/stickyheadersrecyclerview/StickyRecyclerHeadersTouchListener.java [hunk 1]
```diff
@@ -1,61 +1,66 @@
 package com.timehop.stickyheadersrecyclerview;
 
+import android.media.SoundPool;
```
Comment (jacobtabak) [inline library/src/main/java/com/timehop/stickyheadersrecyclerview/StickyRecyclerHeadersTouchListener.java]: Unused import

File: library/src/main/java/com/timehop/stickyheadersrecyclerview/StickyRecyclerHeadersTouchListener.java [hunk 2]
```diff
@@ -1,61 +1,66 @@
...
 import android.support.v7.widget.RecyclerView;
 import android.view.GestureDetector;
 import android.view.MotionEvent;
+import android.view.SoundEffectConstants;
 import android.view.View;
+import android.view.accessibility.AccessibilityEvent;
+import android.widget.Button;
```
Comment (jacobtabak) [inline library/src/main/java/com/timehop/stickyheadersrecyclerview/StickyRecyclerHeadersTouchListener.java]: AccessibilityEvent and Button are both unused imports

**Comment by jacobtabak** [comment]
Please adjust spacing to conform with the rest of the project. 2 spaces for tab

File: library/src/main/java/com/timehop/stickyheadersrecyclerview/StickyRecyclerHeadersTouchListener.java [hunk 3]
```diff
@@ -1,61 +1,66 @@
...
+        public void onHeaderClick(View header, int position, long headerId);
+    }
+
+    public StickyRecyclerHeadersTouchListener(final RecyclerView recyclerView,
+                                              final StickyRecyclerHeadersDecoration decor) {
+        mTapDetector = new GestureDetector(recyclerView.getContext(), new SingleTapDetector());
+        mRecyclerView = recyclerView;
+        mDecor = decor;
     }
-  }
```
Comment (jacobtabak) [inline library/src/main/java/com/timehop/stickyheadersrecyclerview/StickyRecyclerHeadersTouchListener.java]: Extra space - guess this was here before, but would you mind fixing?

**Comment by jacobtabak** [comment]
Thanks for the contribution! Looking forward to merging.

**Comment by forcewill** [comment]
Done hopefully its all correct now

**Comment by jacobtabak** [comment]
Thanks! I did a manual merge to hammer out a few more spacing issues.

**Pull request merged**